demo/admin/src/products/generator/generated/ProductForm.tsx

```diff
@@ -209,16 +209,10 @@ export function ProductForm({ id }: FormProps) {
                                     name
                                 }
                             }
-                        }`, variables: { filter: { addressAsEmbeddable_country: { equal: values.type } } }
+                        }`, variables: { filter: { addressAsEmbeddable_country: { equal: manufacturerCountry } } }
                 });
                 return data.manufacturers.nodes;
-            }} getOptionLabel={(option) => option.name} disabled={!values?.type}/><OnChangeField name="type">
-                            {(value, previousValue) => {
-                if (value.id !== previousValue.id) {
-                    form.change("manufacturer", undefined);
-                }
-            }}
-                        </OnChangeField>
+            }} getOptionLabel={(option) => option.name}/>
```

Comment:
The other option where the filter is another field is now removed from Demo. Could we have both?